docs/architecture/smart-rollups.mdx

```diff
@@ -1,266 +1,178 @@
 ---
-title: Smart Optimistic Rollups
-authors: 'Nomadic Labs, TriliTech'
+title: Smart Rollups
+authors: 'Nomadic Labs, TriliTech, Tim McMackin'
 last_update:
-  date: 30 June 2023
+  date: 4 Jan 2024
 ---
 
-Rollups play a crucial part in providing next-generation scaling on Tezos. This page gives a technical introduction to Smart Rollups, their optimistic nature, and an intro to developing your own WASM kernel.
+import LucidDiagram from '@site/src/components/LucidDiagram';
 
-## Examples
+Smart Rollups play a crucial part in providing high scalability on Tezos.
+They handle transactions and logic in a separate environment that doesn't need to follow all of the same rules as the main Tezos network.
+For this reason, they are not subject to the same block delays and data space constraints as the main Tezos network.
 
-For examples of Smart Rollups, see this repository: https://gitlab.com/tezos/kernel-gallery.
+The transactions and logic that Smart Rollups run is called _layer 2_ to differentiate it from the main network, which is called _layer 1_.
+
+The compiled code of a Smart Rollup must be provided publicly, so that anyone can run a Smart Rollup node to execute that code and verify that other nodes are running it correctly, just like anyone can run nodes, bakers, and accusers on layer 1.
+This code, called the _kernel_, must behave in a deterministic manner and according to a given semantics, which guarantees that results are reproducible by any rollup node with the same kernel.
+The semantics is precisely defined by a reference virtual machine called a proof-generating virtual machine (PVM), able to generate a proof that executing a program in a given context results in a given state.
+This guarantees that if a divergence in results is found, it can be tracked down to a single elementary step that was not executed correctly by some node.
+In this way, multiple nodes can run the same rollup, and each node can verify the state of the rollup.
+
+For a tutorial on Smart Rollups, see [Deploy a Smart Rollup](../tutorials/smart-rollup).
+
+For reference on Smart Rollups, see [Smart Optimistic Rollups](https://tezos.gitlab.io/active/smart_rollups.html) in the Octez documentation.
+
+This diagram shows a high-level view of how Smart Rollups interact with layer 1:
+
+<LucidDiagram width="640px" height="480px" src="https://lucid.app/documents/embedded/3953825e-e5aa-43bb-bc43-701bb0ac3b73" id="stUZbULq.w1v" />
+
+## Uses for Smart Rollups
+
+- Smart Rollups allow you to run large amounts of processing and manipulate large amounts of data that would be too slow or expensive to run on layer 1.
+
+- Smart Rollups are not constrained to the layer 1 block baking rate and therefore can run far more transactions per second than layer 1.
```

Comment:
This is not entirely true. Without a "sequencer",  a rollup receives input by following L1 blocks and hence are subject to the same baking rate.

Reply:
But they can run a lot of logic in between blocks, so they aren't constrained by putting every transaction into a block. Is there a way to explain how/why L2 nodes can run "far more TPS" or should I simplify this to say "Smart Rollups can run far more transactions per second..."?

Reply:
Changed to "Smart Rollups can run far more transactions per second than layer 1."